### PR TITLE
feat: add about us into docs

### PR DESCRIPTION
**Pull request by tuledu**
Could all of you comment on your background in my PR for updating it in docs? @saigon-technology/react-native

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,47 @@ This project is a template for [React Native](https://reactnative.dev/) that can
...
+Welcome to Awesome App! We're a small team of developers passionate about creating library that make people's lives easier.
+
+### Team Members
+
+Our team consists of 14 developers with diverse backgrounds:
+
+- Hong Hoang: Our lead developer with over 10 years of experience in software development and 5 years of experience in mobile application (react native). He's responsible for the overall architecture and technical implementation of the app.
+- Chung Nguyen: 
+- Phi Tran: 
+- Vu Nguyen:
```
Comment (vunguyendhSTS) [inline README.md:32]: Over 4 years experience in software development and mobile application (React Native).

File: README.md [hunk 2]
```diff
@@ -110,5 +151,38 @@ Thanks goes to these wonderful people
...
+
+## Networking
+
+- [Networking](/docs/networking.md)
+
+## Contributing
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
```
Comment (loc-nguyenthien) [inline README.md:186]: I think we removed these sections above, they are replace with more complete contents

File: README.md [hunk 3]
```diff
@@ -18,6 +18,47 @@ This project is a template for [React Native](https://reactnative.dev/) that can
...
+Our team consists of 14 developers with diverse backgrounds:
+
+- Hong Hoang: Our lead developer with over 10 years of experience in software development and 5 years of experience in mobile application (react native). He's responsible for the overall architecture and technical implementation of the app.
+- Chung Nguyen: 
+- Phi Tran: 
+- Vu Nguyen: 
+- Huy Duong: 
+- Anh Le Hoang: 
+- Thinh Vo: 
+- Loc Nguyen:
```
Comment (loc-nguyenthien) [inline README.md:36]: with 5 years working in Front-end Development and over 4 years experience with Mobile Development (React Native).

File: README.md [hunk 4]
```diff
@@ -18,6 +18,47 @@ This project is a template for [React Native](https://reactnative.dev/) that can
 
 NodeJS (version 16 or later) and NPM is required.
 To make sure you have them available on your machine, try running the following command
+## About Us
+
+Welcome to Awesome App! We're a small team of developers passionate about creating library that make people's lives easier.
```
Comment (loc-nguyenthien) [inline README.md:23]: How about `Welcome to Awesome App! We're a small team of developers passionate at making our own developer lives easier.`

**Pull request closed without merging**